static/app/views/performance/queues/destinationSummary/transactionsTable.tsx

```diff
@@ -0,0 +1,213 @@
+import {Fragment} from 'react';
+import {browserHistory, Link} from 'react-router';
+import styled from '@emotion/styled';
+import type {Location} from 'history';
+import qs from 'qs';
+
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  type GridColumnHeader,
+} from 'sentry/components/gridEditable';
+import type {CursorHandler} from 'sentry/components/pagination';
+import Pagination from 'sentry/components/pagination';
+import {t} from 'sentry/locale';
+import type {Organization} from 'sentry/types';
+import type {EventsMetaType} from 'sentry/utils/discover/eventView';
+import {getFieldRenderer} from 'sentry/utils/discover/fieldRenderers';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+import {renderHeadCell} from 'sentry/views/starfish/components/tableCells/renderHeadCell';
+import {useSpanMetrics} from 'sentry/views/starfish/queries/useSpanMetrics';
+import type {MetricsResponse} from 'sentry/views/starfish/types';
+import {QueryParameterNames} from 'sentry/views/starfish/views/queryParameters';
+
+type Row = Pick<
+  MetricsResponse,
+  | 'avg_if(span.self_time,span.op,queue.task.celery)'
```

Comment:
👀 this is interesting, why? Do queue span ops not map cleanly to a module? Why is the average `avg_if`, but the `sum` isn't a `sum_if`?

Reply:
So right now `queue.task.celery` ops refer to consumer spans and `queue.submit.celery` refers to producer spans (for celery).
 
One thing is, I haven't added `queue` spans as a known module yet in relay, which is why I'm not able to query by `span.module`. 

The other thing is that for this query, I'm filtering this query to include both `task` and `submit` ops, but i just want the average duration of the `task` ops as a column. The `sum` in this query is purposefully the total duration sum across both `task` and `submit` which is why I don't need any conditional `sum_if` here (although this sum will probably change because it's not really conveying what I'm looking for either. That's a different topic).

So basically this is because I'm still experimenting with queue spans in ui and relay at the moment 😄

Reply:
Roger